Dockerfile

```diff
@@ -2,31 +2,49 @@
 FROM node:22 AS frontend-build
 WORKDIR /frontend
 
-COPY app/frontend/ /frontend/
+COPY frontend/ /frontend/
 RUN npm install
 RUN npm run build
 
-# Stage 2 - copy files, build and serve python app
-FROM python:3.13.2-slim
+# Stage 2 - build and package Python app
+FROM python:3.13.2-slim AS build
 WORKDIR /app
-COPY --from=frontend-build /frontend/dist /app/app/frontend/dist
 
-COPY requirements.txt /app/app/
-COPY app/*.py app/*.json /app/app/
-COPY example_config_combined.yaml /app/
+# Copy and install requirements
+COPY pyproject.toml /app/
+COPY src/ /app/src/
+
+# Copy frontend build
+COPY --from=frontend-build /frontend/dist /app/src/script_runner/frontend/dist
 
-COPY examples/scripts/ /app/examples/scripts/
-COPY examples/requirements.txt /app/examples/
+# Build the wheel
+RUN pip install build
+RUN python -m build --wheel
```

Comment:
not sure i follow why we build a wheel in the dockerfile here -- wouldn't that be done in a build/publish/release workflow following the craft process (https://github.com/getsentry/craft)